### PR TITLE
View Favourites

### PR DESCRIPTION
**Pull request by jrdiazr**
Feature, View favorites connected with API and details page.

### CODE REVIEW

File: liveHomeFrontend/pages/favourite/[propertyId].js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  //       await API.setLikeProperty(propertyId, user.userId, token);
+  //     }
+  //     const newPost = [...post];
+  //     newPost.find(item => item.id === propertyId).favorites = !newPost.find(item => item.id === propertyId).favorites;
+  //     setPost(newPost);
+  //   } catch (error) {
+  //     console.error('[error]', error);
+  //     setError(error);
+  //   }
+  // };
```
Comment (helbertoro) [inline liveHomeFrontend/pages/favourite/[propertyId].js:76]: delete coments

File: liveHomeFrontend/pages/post/new_post_step_three.js [hunk 2]
```diff
@@ -48,7 +48,29 @@ const newPostStepThree = () => {
...
+            <Checkbox name='warehouse' {...warehouse} text='Warehouse' />
+          </FormField>
+          <FormField>
+            <Input type='number' label='Parkig' name='parkng' required {...parking} />
+          </FormField>
+          <FormField>
+            <TextareaComponent label='Nearby places' name='nearby_places' required {...nearby_places} />
+=======
             <Button value='Upload image' buttonClass='redLinearButton' buttonType='button' />
+>>>>>>> d2f69daabf5b48393cd4c92cd2e6ba3fc4607133
```
Comment (helbertoro) [inline liveHomeFrontend/pages/post/new_post_step_three.js:73]: there is a mistake

File: liveHomeFrontend/pages/post/new_post_step_three.js [hunk 3]
```diff
@@ -60,7 +82,11 @@ const newPostStepThree = () => {
           </div>
           <div className={styles.buttons}>
             <Button value='Back' buttonClass='grayLightLinearButton' buttonType='button' handleClick={() => { Router.back(); }} />
+<<<<<<< HEAD
+            <Button value='Continue' buttonClass='greenLinearButton' buttonType='submit' />
+=======
             <Button value='Create post' buttonClass='greenButton' buttonType='button' handleClick={(e) => handleSubmit(e)} />
+>>>>>>> d2f69daabf5b48393cd4c92cd2e6ba3fc4607133
```
Comment (helbertoro) [inline liveHomeFrontend/pages/post/new_post_step_three.js:89]: mistake

File: liveHomeFrontend/src/utils/api.js [hunk 4]
```diff
@@ -3,7 +3,6 @@ import fetch from 'node-fetch';
 import FormData from 'form-data';
 const API_URL = 'https://live-home.now.sh/api';
 
-
 class API {
   async getAccount (userId, token) {
     try {
```
Comment (helbertoro) [inline liveHomeFrontend/src/utils/api.js:8]: This try is empty?

Comment (jrdiazr) [reply liveHomeFrontend/src/utils/api.js:8]: this error was resolve in next commit

**Pull request merged**